modules/src/ics24_host/validate.rs

```diff
@@ -9,6 +9,7 @@ macro_rules! bail {
 
 /// Path separator (ie. forward slash '/')
 const PATH_SEPARATOR: char = '/';
+const VALID_SPECIAL_CHARS: &str = "._+-# []<>";
```

Comment:
In the ics24 spec, it seems a 'space' is not a valid char. Does that space between '#' and '[' cause some issues ? For example a identifier like 'abc xyz' would be considered valid by the validate_identifier function.

Reply:
Yup that blank space is an intruder. That's a very good catch!